_dashboards/visualize/gantt.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Using Gantt charts
+parent: Visualization types
+grand_parent: Visualize
+nav_order: 25
+redirect_from:
+  - /dashboards/gantt/
+---
+
+# Using Gantt charts
+
+Gantt charts are a visualization type that displays events, steps, and tasks as horizontal bars. The length of the bars shows the amount of time associated with that event, step, or task. Gantt charts are used to represent a series of events that contain a parent-child relationship. This can be particularly useful in trace analytics, telemetry, and monitoring use cases, in which the users need to understand the overall interaction between traces or events.
```

Comment:
```suggestion
A Gantt chart is a visualization type that displays events, steps, and tasks as horizontal bars. The length of the bars shows the amount of time associated with that event, step, or task. Gantt charts are used to represent a series of events that contain a parent-child relationship. This can be particularly useful in trace analytics, telemetry, and monitoring use cases, in which the user needs to understand the interactions between traces and events.
```